mcp/streamable.go

```diff
@@ -1389,6 +1389,20 @@ type StreamableClientTransport struct {
 	// It defaults to 5. To disable retries, use a negative number.
 	MaxRetries int
 
+	// DisableListening disables receiving server-to-client notifications when no request is in flight.
```

Comment:
Should we just call this "DisableStandaloneSSE" to be clear? What do other SDKs call it, if they have such a setting?